src/tls/keypair.h

```diff
@@ -860,6 +872,20 @@ namespace tls
       if (mbedtls_x509write_crt_set_authority_key_identifier(&sign.crt) != 0)
         return {};
 
+      // Because mbedtls does not support parsing x509v3 extensions from a
```

Comment:
Will this cause problems when we switch to the network generating a TBS for the quote?

Reply:
I don't think it will but we should bear it in mind going forward.